libs/solid/skeleton/src/lib/services/feedback.service.ts

```diff
@@ -35,19 +35,17 @@ export class FeedbackService {
       location: location,
       title: location ? 'Fehler melden' : 'Kontakt und Feedback',
       subject: location ? 'Fehler melden' : 'Feedback',
-      message: location ? '' + '\n\nFehler gefunden in: ' + location : ''
+      message: location ? '' + '\n\nFehler gefunden in: ' + location : '',
```

Comment:
Wie Du vielleicht gesehen hast, haben wir jetzt das Problem, dass das jetzt immer in der `message` steht :wink:

Ein Vorschlag: Es könnte wirklich erst beim Senden hinzugefügt werden, also nicht in der FeedbackComponent, sondern direkt in `submitFeedback(value: any)` (im feedback.service).
so in der Art von `value.message += '\n\n' + location`

Reply:
Das finde ich auch schön

Reply:
Das fehlt jetzt noch, oder?

Reply:
Das hier ist der letzte Baustein, dann ist es fertig!

Reply:
![Screenshot (52)](https://user-images.githubusercontent.com/33594579/158055036-bf9e0a6f-cc50-4917-b199-6dde1cc73018.png)

In der FeedbackComponent habe ich so gemacht. Was denkst du ?

Reply:
Ja, das ist noch besser, als im `feedback.service.ts` !

Ich würde sagen, wir sind fertig 😃